### DISCUSSION TITLE
Updated ITPA confinement scalings: ITPA20

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#2889] [Ideas]
In GitLab by @mkovari on Jul 9, 2018, 11:36

The analysis of the updated energy confinement scaling is now published:  
[The updated ITPA global H-mode confinement database: description and analysis](https://iopscience.iop.org/article/10.1088/1741-4326/abdb91/meta)  
G Verdoolaege et al

>  Extrapolation of this scaling to ITER yields a somewhat lower confinement time compared to the IPB98(y,2) prediction.

The paper also includes regression results for individual machines, including MAST and NSTX, which may be helpful for ST work.

The engineering scaling, henceforth referred to as ITPA20 is  
![image](https://github.com/ukaea/PROCESS/assets/9718416/30f2b127-6794-4455-b83e-6ef1ac8a1460)

This is based on the entire set of ELMy H-modes in the database, not restricted to the ITER-like subset.

There is also another scaling expressed in dimensionless variables, not intended for the purpose of prediction, denoted ITPA20-
IL-dim, using only ITER-like data.

Mikhail Maslov is one of the authors.  

Note that Mikhail also has very important data on the dependence of confinement on the separatrix density $`n_{e,sep}`$

Suggestions
*  Update PROCESS.
*  Investigate what the updated database says about low aspect ratio machines.
*  Investigate what the database can tell us about the dependence of confinement on the separatrix density.
 

@stuartmuldrew @jmorris-uk @ajpearcey @cjwgriesel @dshort

**Comment by mkovari** [comment]
Note that this is also discussed in #1852, where some but not all of this has been implemented.

**Comment by mkovari** [comment]
Now implemented #3164.
